tests/conftest.py

```diff
@@ -14,14 +14,19 @@
 
 
 REDIS_INFO = {}
-default_redis_url = "redis://localhost:6379/9"
+DEFAULT_REDIS_URL = "redis://localhost:6379/9"
+DEFAULT_REDIS_MASTER_HOST = "localhost"
 
 
 def pytest_addoption(parser):
-    parser.addoption('--redis-url', default=default_redis_url,
+    parser.addoption('--redis-url', default=DEFAULT_REDIS_URL,
                      action="store",
                      help="Redis connection string,"
                           " defaults to `%(default)s`")
+    parser.addoption('--redis-master-host', default=DEFAULT_REDIS_MASTER_HOST,
```

Comment:
Is there ever a reason where `redis-master-host` would not point to the same host that's defined in the `redis-url`? I can't think of one.

Assuming there isn't a valid reason, we should infer the master hostname from the `redis-url` argument rather than introducing a new setting.

Reply:
Probably not. Fixed!